src/Nest/CommonAbstractions/Infer/ForecastIds/ForecastIds.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	[DebuggerDisplay("{DebugDisplay,nq}")]
+	public class ForecastIds : IUrlParameter, IEquatable<ForecastIds>
+	{
+		private readonly List<string> _forecastIds;
+
+		public ForecastIds(IEnumerable<string> forecastIds) => _forecastIds = forecastIds?.ToList() ?? new List<string>();
+
+		public ForecastIds(string forecastIds) => _forecastIds = forecastIds.IsNullOrEmpty()
+			? new List<string>()
+			: forecastIds.Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries)
+				.Select(s => s.Trim())
+				.ToList();
+
+		internal IReadOnlyList<string> Ids => _forecastIds;
```

Comment:
I don't think this is necessary. But if it is, I think making the property type `List<string>` would be preferable to accessing through the interface for an internal property.

Reply:
Required for the equality comparison.